### PR TITLE
tests/manual: add in coreos-builds-bisect script

### PR DESCRIPTION
**Pull request by dustymabe**
This script will allow us to perform bisection on our list of builds for our CoreOS streams. It will enable us to find issues faster. It was inspired by rpm-ostree-bisect [1], which does somethign similar, but operates on the OSTree repo from inside the OS.

Example output looks something like:

```
+ echo 'Could not access /etc/shadow -> SUCCESS'
Could not access /etc/shadow -> SUCCESS
+ exit 0
38.20230819.3.0 passed
BISECT TEST RESULTS:
Last known good build: 38.20230819.3.0
First known bad build: 38.20230902.3.0
```

[1] https://github.com/ostreedev/ostree-releng-scripts/blob/master/rpm-ostree-bisecthis

### CODE REVIEW

**Review by jlebon — APPROVED** [review]
I didn't review the script in depth, but it looks sane to me!

Though I wonder if it should be in e.g. `tests/tools/` or `tests/scripts/` instead? All the other files in `tests/manual/` currently are actual scripts that test specific features and so are accurately described by the directory name. This seems like a more generic tool?

**Comment by dustymabe** [comment]
> Though I wonder if it should be in e.g. `tests/tools/` or `tests/scripts/` instead? All the other files in `tests/manual/` currently are actual scripts that test specific features and so are accurately described by the directory name. This seems like a more generic tool?

I don't have a strong opinion there. Mostly the things in here are here because we didn't have a better place to put them and you generally know what you are looking for if you get down this far..

With as few of these that there are I'm not sure making other directories will add value (i.e. compared to the more searching) until we get into maybe 10s of files?

TL;DR Want it in a `scripts/` dir? I'll put it there if you think it's worth it.

**Comment by jlebon** [comment]
> TL;DR Want it in a `scripts/` dir? I'll put it there if you think it's worth it.

Actually yeah, `scripts/` is even more appropriate. But no strong opinion either if you want to keep it there.

**Comment by dustymabe** [comment]
I'll keep it where it is for now just to save a CI cycle. Thanks for the review!
